features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/routing/common.jag

```diff
@@ -0,0 +1,17 @@
+<%
+var log = new Log("Common Router");
+
+var uri = request.getRequestURI();
+var filePath = uri.split(request.getContextPath());
+
+log.info(uri.split(request.getContextPath()));
```

Comment:
Does this need to be logged as  a info log? If so please append required identifier.